Makefile

```diff
@@ -68,14 +69,10 @@ build-requirements:
 	$(TEMPDIR)/bin/pip freeze > requirements.txt
 
 update-kinto-admin:
-	rm -fr kinto/plugins/admin/static/*
-	rm -fr tmp
-	git clone https://github.com/Kinto/kinto-admin.git tmp
-	cd tmp; git checkout $$(git tag | tail -n 1); npm install
-	cd tmp; KINTO_ADMIN_PUBLIC_PATH=/v1/admin/ npm run dist
-	cp -fr tmp/dist/* kinto/plugins/admin/static/
-	sed -i "s/ version=\".*\"/ version=\"$$(tmp/bin/kinto-admin --version)\"/g" kinto/plugins/admin/__init__.py
-	rm -fr tmp
+	rm -fr kinto/plugins/admin/build
+	rm -fr kinto/plugins/admin/node_modules
```

Comment:
It was to install the last version of the dependencies. In anycase I don't like to have a node_modules folder in a Python software tree 🗡